doc/source/dev/api-dev/array_api.rst

```diff
@@ -90,13 +90,20 @@ More strict array input validation will reject ``np.matrix`` and
 Currently supported functionality
 `````````````````````````````````
 
-The following modules provide Array API standard support when the environment
+The following modules provide array API standard support when the environment
 variable is set:
 
 - ``scipy.cluster.hierarchy``
 - ``scipy.cluster.vq``
 - ``scipy.fft``
 
+Partial support is provided in ``scipy.special`` for the following functions:
```

Comment:
I think all of these should be single backticks so we get a hyperlink.

I'm not sure whether "partial" describes support for `special` or the functions themselves. As written, it looks like it is the functions. This is technically true in the sense that we don't support their ufuncness, but for practical purposes, but I'm not sure if that's worth mentioning.

I'd just drop "partial". It's clear that not every function in special is supported since only 14 are listed.

Reply:
addressed 👍